Main.py

```diff
@@ -345,11 +345,11 @@ def precollect_hint(location: Location, auto_status: HintStatus):
                 for key in ("slot_data", "er_hint_data"):
                     multidata[key] = convert_to_base_types(multidata[key])
 
-                multidata = zlib.compress(restricted_dumps(multidata), 9)
+                serialized_multidata = zlib.compress(restricted_dumps(multidata), 9)
```

Comment:
```suggestion
                serialized_multidata: bytes = zlib.compress(restricted_dumps(multidata), 9)
```

Reply:
Our style guide says "where appropriate", which, on it's own, would be a little vague. But it elaborates a little.
```
* Use type annotations where possible for function signatures and class members.
* Use type annotations where appropriate for local variables (e.g. `var: List[int] = []`, or when the
  type is hard or impossible to deduce.) Clear annotations help developers look up and validate API calls.
```